src/components/results/results.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import DoctorInfoList from './doctorInfoList.js';
+import TotalResults from './totalResults.js';
+
+
+
+class Results extends React.Component {
+
+    constructor(props) {
+      super(props);
+      this.state = {      
+          data: this.props.resultsData.results  
+      };  
+    }
+
+    ccomponentDidMount() {
+      console.log('component life cycles will go here, this is just an example')
```

Comment:
[question/fix] Are you planning to use this for something? If not, it can be deleted. As far as your comment goes, `componentDidMount` **is** a life cycle method. I just want to make sure you're understanding the terminology and how to work with life cycle methods.